tests/test_checks.py

```diff
@@ -120,7 +123,17 @@ def test_panels_is_empty(self):
                     hint="Set DEBUG_TOOLBAR_PANELS to a non-empty list in your "
                     "settings.py.",
                     id="debug_toolbar.W005",
-                )
+                ),
+                Error(
+                    "The Django Debug Toolbar can't be used with tests",
+                    hint="Django changes the DEBUG setting to False when running "
+                    "tests. By default the Django Debug Toolbar is installed because "
+                    "DEBUG is set to True. For most cases, you need to avoid installing "
+                    "the toolbar when running tests. If you feel this check is in error, "
+                    "you can set `DEBUG_TOOLBAR_CONFIG['IS_RUNNING_TESTS'] = False` to "
+                    "bypass this check.",
+                    id="debug_toolbar.W008",
+                ),
```

Comment:
Why is `run_checks` returning this error? I wouldn't have expected this test to require a change.

Reply:
this is what I was getting 

```
FAIL: test_panels_is_empty (tests.test_checks.ChecksTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "django-debug-toolbar/tests/test_checks.py", line 98, in test_panels_is_empty
    self.assertEqual(
AssertionError: Lists differ: [<War[165 chars]005'>, <Error: level=40, msg="The Django Debug[413 chars]08'>] != [<War[165 chars]005'>]

First list contains 1 additional elements.
First extra element 1:
<Error: level=40, msg="The Django Debug Toolbar can't be used with tests", hint="Django changes the DEBUG setting to False when running tests. By default the Django Debug Toolbar is installed because DEBUG is set to True. For most cases, you need to avoid installing the toolbar when running tests. If you feel this check is in error, you can set `DEBUG_TOOLBAR_CONFIG['IS_RUNNING_TESTS'] = False` to bypass this check.", obj=None, id='debug_toolbar.W008'>

Diff is 1179 characters long. Set self.maxDiff to None to see it.
```

Reply:
Why is the test settings `IS_RUNNING_TESTS` value of `False` not being respected?

Reply:
I am working on this to see what the issue is.